test/e2e-tests-yaml.sh

```diff
@@ -30,7 +30,7 @@ delete_files() {
 
 main() {
   versions="v1alpha1 v1beta1"
-  examples="triggertemplates triggerbindings eventlisteners clustertriggerbindings"
+  examples="triggertemplates triggerbindings eventlisteners clustertriggerbindings github-owners"
   for v in ${versions}; do
```

Comment:
Please remove `github-owners` from this. It doesn't exist  in v1alpha1

Reply:
addressed